src/ecmascript_simd.js

```diff
@@ -474,13 +474,13 @@ if (typeof SIMD.Bool32x4.select === "undefined") {
   }
 }
 
-if (!Object.hasOwnProperty(SIMD.Bool32x4.prototype, 'valueOf')) {
+if (!SIMD.Bool32x4.prototype.hasOwnProperty('valueOf')) {
```

Comment:
`Object.prototype.hasOwnProperty.call` would be better here, in case someone adds a `hasOwnProperty` function to one of the types.